README.md

```diff
@@ -47,22 +47,40 @@ Furthermore, if you have an **algorithm proposition**, please fill in the survey
 ### Community
 Make sure to check out Memgraph community, and join us on a survey of streaming a graph algorithms! Drop us a message on the channels below:
 
-- :robot: [**Discord**](https://discord.gg/memgraph)
-- :busts_in_silhouette: [**Discourse forum**](https://discourse.memgraph.com/)
-- :octocat: [**Memgraph GitHub**](https://github.com/memgraph)
-- :bird: [**Twitter**](https://twitter.com/memgraphdb)
-- :movie_camera: [**YouTube**](https://www.youtube.com/channel/UCZ3HOJvHGxtQ_JHxOselBYg)
+<p align="center">
+<a href="https://twitter.com/intent/follow?screen_name=memgraphmage">
+    <img src="https://img.shields.io/badge/@memgraphmage-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="Follow @memgraphmage"/>
+  </a>
+<a href="https://discourse.memgraph.com/">
+    <img src="https://img.shields.io/badge/Discourse_forum-000000?style=for-the-badge&logo=discourse&logoColor=white" alt="Discourse forum"/>
+</a>
+<a href="https://memgr.ph/join-discord">
+    <img src="https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white" alt="Discord"/>
+</a>
+<a href="https://github.com/memgraph">
+    <img src="https://img.shields.io/badge/Memgraph_GitHub-181717?style=for-the-badge&logo=github&logoColor=white" alt="Memgraph Github"/>
+</a>
+<a href="https://www.youtube.com/channel/UCZ3HOJvHGxtQ_JHxOselBYg">
+    <img src="https://img.shields.io/badge/YouTube-FF0000?style=for-the-badge&logo=youtube&logoColor=white" alt="Memgraph YouTube"/>
+</a>
+</p>
 
 ## Overview
-- [Compatibility](#memgraph-compatibility)
-- [Installation](#how-to-install)
-- [Example](#running-the-mage)
-- [MAGE Spells](#mage-spells)
-- [Advanced configuration](#advanced-configuration)
-- [Developing](#developing-mage-with-docker)
-- [Testing](#testing-the-mage)
-- [Contributing](#contributing)
-- [Code of Conduct](#code-of-conduct)
+- [Memgraph compatibility](#memgraph-compatibility)
+  - [How to install?](#how-to-install)
+  - [Further steps - Installation](#further-steps---installation)
+    - [1. Installing MAGE with Docker](#1-installing-mage-with-docker)
+      - [a) Install MAGE from Docker Hub](#a-install-mage-from-docker-hub)
+      - [b) Install MAGE with Docker build of repository](#b-install-mage-with-docker-build-of-repository)
+    - [2. Installing MAGE on Linux distro from source](#2-installing-mage-on-linux-distro-from-source)
+      - [Prerequisites](#prerequisites)
+  - [Running the MAGE](#running-the-mage)
+  - [MAGE Spells](#mage-spells)
+  - [Advanced configuration](#advanced-configuration)
+  - [Testing the MAGE](#testing-the-mage)
+  - [Contributing](#contributing)
+  - [Code of Conduct](#code-of-conduct)
+  - [Feedback](#feedback)
```

Comment:
Check formatting here, I think this is indented too much.

Reply:
I just think we can put `How to install`, `Running the MAGE`, and so on in the same indent, that seems like it belongs to the same scope.